recipes/projectcard/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "projectcard" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/projectcard-{{ version }}.tar.gz
+  sha256: 7458808f4ca6af0502ff4244dead29426c266005f7666dac6780dfad024c8e48
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=60
+    - setuptools-scm >=8.0
+    - wheel
+    - pip
+  run:
+    - python >=3.6
+    - flake8
+    - jsonref
+    - jsonschema
+    - pydantic >=2.0
+    - pyyaml
+    - toml
+
+test:
+  imports:
+    - projectcard
+  commands:
```

Comment:
It looks like there are two cli tools. Could you add tests for them:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands
https://github.com/network-wrangler/projectcard/blob/main/pyproject.toml#L37